### ISSUE TITLE
OpenAI Library is sync and blocking while Discord.py is asynchronous

### ISSUE THREAD

**Issue by applebee1558** [openai/gpt-discord-bot#34]
Unless I'm getting something wrong, shouldn't the openai function calls be within thread executors so it doesn't block the main bot thread? A blocking library would prevent the async event loop (what discord.py uses) from processing anything while it is blocking.

**Comment by YufeiG** [comment]
yes, we recently added async to our python sdk. I need to move over to them, thanks for the reminder!

**Comment by Kav-K** [comment]
This is fixed!
